src/dex-ui/utils/poolMetrics.ts

```diff
@@ -185,6 +186,22 @@ const isHbarToken = (TokenID: string): boolean => {
   return TokenID === HBARTokenId;
 };
 
+interface getAllPoolTransactionFeeParams {
+  tokenPairs: TokenPair[] | null;
+  poolSymbol: string | undefined;
+}
+
+const getAllPoolTransactionFee = (params: getAllPoolTransactionFeeParams) => {
+  return (
+    params.tokenPairs
+      ?.filter((pair) => pair?.lpTokenMeta.symbol === params.poolSymbol)
```

Comment:
Pairs should be matched by pair ids (Hedera Account Id / address). Text symbols are not guaranteed to be unique.